modules/photons_canvas/addon.py

```diff
@@ -77,7 +77,6 @@ async def execute_task(self, **kwargs):
 
         if self.reference in register.available_animations():
             ref = self.artifact
-            artifact = self.reference
```

Comment:
This should probably be more `self.artifact = self.reference`

I'll have a look later :)